lib/components/AlbumScreen/album_screen_content.dart

```diff
@@ -170,10 +170,10 @@ class _AlbumScreenContentState extends ConsumerState<AlbumScreenContent> {
                 childrenForQueue: queueChildren,
                 parent: widget.parent,
                 onRemoveFromList: onDelete,
-                showDateAdded: (widget.parent.type == "Playlist" && playlistSortBy == SortBy.dateCreated),
-                showPlayCount: (widget.parent.type == "Playlist" && playlistSortBy == SortBy.playCount),
-                showDateLastPlayed: (widget.parent.type == "Playlist" && playlistSortBy == SortBy.datePlayed),
-                showReleaseDate: (widget.parent.type == "Playlist" && playlistSortBy == SortBy.premiereDate),
+                showDateAddedAdaptive: (widget.parent.type == "Playlist" && playlistSortBy == SortBy.dateCreated),
+                showPlayCountAdaptive: (widget.parent.type == "Playlist" && playlistSortBy == SortBy.playCount),
+                showDateLastPlayedAdaptive: (widget.parent.type == "Playlist" && playlistSortBy == SortBy.datePlayed),
+                showReleaseDateAdaptive: (widget.parent.type == "Playlist" && playlistSortBy == SortBy.premiereDate),
```

Comment:
how about passing a single enum here instead, that decides which info is shown? I guess we'll only ever show one extra field.  

```suggestion
                showExtraInfo: widget.parent.type == "Playlist" ? playlistSortBy : null,
```

Reply:
should've caught that last time, but I think it would simplify the code quite a bit

Reply:
Yes, good idea... I left it open so that we maybe later can decide to show multiple, but I guess then we could just undo it, and it definitely reduces code lines :P